wdl_runner/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
-FROM java:openjdk-8-jre
+FROM openjdk:8-jre
```

Comment:
My understanding is that Oracle will discontinue maintenance updates on their distribution of OpenJDK 8 in April 2019.

A recommended solution I've seen is https://hub.docker.com/r/adoptopenjdk/openjdk8 which is an independent distribution of OpenJDK that will continue receiving updates.

Reply:
I changed to openjdk based on the verbiage from the original [java](https://hub.docker.com/_/java)

> This image is officially deprecated in favor of the openjdk image